phpcs.xml.dist

```diff
@@ -115,6 +115,8 @@
 				<element value="apply_block_core_search_border_style"/>
 				<element value="apply_block_core_search_border_styles"/>
 				<element value="build_dropdown_script_block_core_categories"/>
+				<element value="build_navigation_link_block_variations"/>
```

Comment:
@kt-12, you are not supposed to add these functions to the ignore list. There is no need to do so; these functions did not exist before the list was created, and there are no backward compatibility issues associated with naming them according to the naming convention implemented in the sniff you're trying to ignore. They could be named `block_core_template_part_build_area_variations()` and `block_core_post_terms_build_variations` respectively. Could you rename them?

Reply:
I added this as I was getting an error from GitHub action. Adding this did resolve the error back then is what I remember, but I am not sure at the moment. Should we remove it?

Reply:
Here is the follow-up #58538.

Reply:
The names I suggested for these functions were not exactly correct, but the idea was to adhere to the coding standard. Thanks for fixing it, @Mamaduka.
I have approved your PR.